app/models/communitarian/resolution_schedule.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module Communitarian
+  class ResolutionSchedule
+    WEEKDAYS = Date::DAYS_INTO_WEEK.keys.freeze
+
+    attr_reader :close_hour, :reopen_delay
+
+    def initialize(close_weekday: self.class.close_weekday,
```

Comment:
do we really need to initialize class with this vars like this? We just call in code above `Communitarian::ResolutionSchedule.new` without sending params.

Reply:
I need it for test :)